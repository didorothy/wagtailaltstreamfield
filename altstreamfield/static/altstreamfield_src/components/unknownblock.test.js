import React from "react";
import { render, fireEvent, wait, waitForDomChange, waitForElementToBeRemoved, cleanup } from "@testing-library/react";
import "@testing-library/jest-dom";
import UnknownBlock from "./unknownblock";


describe('UnknownBlock', () => {
    test("#minimal", () => {
        let container = render(<UnknownBlock block={{value: {one: "1", nest: {list: ['test', 1, 1.0]}}}} blockTypes={[]} onChange={() => {}}/>);
        expect(container.container).toMatchSnapshot();
        cleanup();
    });
});