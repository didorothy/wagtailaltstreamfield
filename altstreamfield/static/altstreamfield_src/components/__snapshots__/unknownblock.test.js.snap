// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UnknownBlock #minimal 1`] = `
<div>
  <div
    class="raw"
  >
    <pre>
      {
  "one": "1",
  "nest": {
    "list": [
      "test",
      1,
      1
    ]
  }
}
    </pre>
  </div>
</div>
`;
